{RepositoryName}/{Namespace}.Web/packages.config

```diff
@@ -4,7 +4,7 @@
   <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net462" />
   <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net462" />
   <package id="MSBuildTasks" version="1.5.0.235" targetFramework="net461" developmentDependency="true" />
-  <package id="Newtonsoft.Json" version="11.0.2" targetFramework="net461" />
+  <package id="Newtonsoft.Json" version="12.0.2" targetFramework="net461" />
   <package id="Unity" version="4.0.1" targetFramework="net462" />
   <package id="VirtoCommerce.Domain" version="{CoreModuleVersion}" targetFramework="net461" />
   <package id="VirtoCommerce.Module" version="0.17.0" targetFramework="net461" developmentDependency="true" />
```

Comment:
update to 0.18.0 as only this version supports VS2019.

Reply:
Fixed